.circleci/config.yml

```
@@ -142,6 +261,38 @@
             export PATH="$HOME/.local/bin:$PATH"
             PYTHONPATH="$CIRCLE_WORKING_DIRECTORY/src" python3 -m pytest --version
             echo "✅ Test framework ready"
+  # Safety scan job - security vulnerability scanning
+  safety-scan:
```

YAMLlint (1.37.1)

[error] duplication of key "safety-scan" in mapping

(key-duplicates)